api/docs/release.dox

```diff
@@ -133,7 +133,23 @@ Dr. Memory Framework (DRMF) in the same package as DynamoRIO.  DRMF
 provides the umbra, drsyscall, and drsymcache Extensions for use by
 clients.
 
-The changes between version \DR_VERSION and 7.1.0 include the following compatibility
+The changes between version \DR_VERSION and 8.0.0 include the following compatibility
+changes:
+
```

Comment:
suggest: I usually put one line `- Nothing yet.` in these sections so it's very clear how to edit.